scalafmt-core/shared/src/main/scala/org/scalafmt/util/GitOps.scala

```diff
@@ -35,15 +35,13 @@ class GitOpsImpl(workingDirectory: AbsoluteFile) extends GitOps {
     }
   }
 
-  override def lsTree: Seq[AbsoluteFile] =
+  override def lsFiles(dir: AbsoluteFile): Seq[AbsoluteFile] =
     Try {
       exec(
         Seq(
           "git",
-          "ls-tree",
-          "-r",
-          "HEAD",
-          "--name-only"
+          "ls-files",
+          dir.path
         )
       ).split(OsSpecific.lineSeparator).toSeq.collect {
         case f if new File(f).isFile => workingDirectory / f
```

Comment:
While you're at it, can you maybe change workingDirectory to rootDir? That should fix https://github.com/scalameta/scalafmt/issues/927